packages/react/src/components/Comments/Thread.tsx

```diff
@@ -74,16 +73,6 @@ export const Thread = ({
     throw new Error("Comments plugin not found");
   }
 
-  const userIds = useMemo(() => {
-    return thread.comments.flatMap((c) => [
-      c.userId,
-      ...c.reactions.flatMap((r) => r.userIds),
-    ]);
-  }, [thread.comments]);
-
-  // load all user data
-  const users = useUsers(editor, userIds);
```

Comment:
ah, I suppose this was a bug because we didn't load the `resolvedBy` userid?

Reply:
Good catch! I only tested resolving the thread on a user that also added a comment to the thread, so I didn't spot this but looks like you're right.